web/api/v1/api_test.go

```diff
@@ -212,7 +212,7 @@ type rulesRetrieverMock struct {
 	testing *testing.T
 }
 
-func (m rulesRetrieverMock) AlertingRules() []*rules.AlertingRule {
+func (m rulesRetrieverMock) AlertingRules(matcherSets ...[]*labels.Matcher) []*rules.AlertingRule {
```

Comment:
Since it doesn't use the parameter.
```suggestion
func (m rulesRetrieverMock) AlertingRules(_ ...[]*labels.Matcher) []*rules.AlertingRule {
```

(even better if the test did use the parameter)

Reply:
remove this parameter as we are not use this parameter at alerts api